lib/containers/ServerSetup.jsx

```diff
@@ -395,6 +411,15 @@ class ServerSetup extends React.PureComponent {
                         {editor}
                     </div>
                     <ConfirmationDialog
+                        show={showingApplyDialog}
+                        onOk={applyServerAgain}
+                        onCancel={hideApplyDialog}
+                        text="You may lose the connections if you apply the server setup again.
```

Comment:
How about: "The device must be reset before applying the server setup. All active Bluetooth connections will be lost. Are you sure you want to apply?"

Reply:
Yes, sounds better ;-)